src/components/externalLinks/ExternalLinksCRUD/DeleteExternalLinkDialog.tsx

```diff
@@ -15,13 +17,20 @@ export default function DeleteExternalLinkDialog({
     const deleteLink = async (): Promise<void> => {
         try {
             setAPICallInProgress(true)
-            const { result } = await deleteExternalLink(selectedLink.id)
+            const { result } = await (isAppConfigView
```

Comment:
can we pass the 2nd param based on the condition instead of writing the whole method?

Reply:
Done